classes/profile_table.php

```diff
@@ -43,18 +43,65 @@ class profile_table extends \table_sql {
         'actions',
     ];
 
-    public function __construct($uniqueid) {
+    const TIME_FORMAT = '%d %b %Y, %H:%M';
```

Comment:
A nice trick is to move time formats into a lang string and then they can be easily customized

```
lang/en/langconfig.php:40:$string['strftimedate'] = '%d %B %Y';
lang/en/langconfig.php:41:$string['strftimedatemonthabbr'] = '%d %b %Y';
lang/en/langconfig.php:42:$string['strftimedatefullshort'] = '%d/%m/%y';
lang/en/langconfig.php:43:$string['strftimedateshort'] = '%d %B';
lang/en/langconfig.php:44:$string['strftimedateshortmonthabbr'] = '%d %b';
lang/en/langconfig.php:45:$string['strftimedatetime'] = '%d %B %Y, %I:%M %p';
lang/en/langconfig.php:46:$string['strftimedatetimeaccurate'] = '%d %B %Y, %I:%M:%S %p';
lang/en/langconfig.php:47:$string['strftimedatetimeshort'] = '%d/%m/%y, %H:%M';
lang/en/langconfig.php:48:$string['strftimedatetimeshortaccurate'] = '%d/%m/%y, %H:%M:%S';
lang/en/langconfig.php:49:$string['strftimedaydate'] = '%A, %d %B %Y';
```